apps/website/src/components/Header.tsx

```diff
@@ -68,13 +71,28 @@ export default function Header() {
 		<header className="sticky top-4 z-20 border border-light-900 rounded-md bg-white/75 shadow backdrop-blur-md dark:border-dark-100 dark:bg-dark-600/75">
 			<div className="block h-16 px-6">
 				<div className="h-full flex flex-row place-content-between place-items-center gap-8">
-					<Button
-						aria-label="Menu"
-						className="h-6 w-6 flex flex-row transform-gpu cursor-pointer select-none appearance-none place-items-center border-0 rounded bg-transparent p-0 text-sm font-semibold leading-none no-underline outline-none lg:hidden active:translate-y-px focus:ring focus:ring-width-2 focus:ring-blurple"
-						onClick={() => setOpened((open) => !open)}
-					>
-						<VscMenu size={24} />
-					</Button>
+					<div className="flex flex-row place-items-center gap-4">
```

Comment:
![image](https://github.com/discordjs/discord.js/assets/20760160/8ca62589-fbb3-4610-84d8-d75397837d7a)

Currently creates a whitespace here, because you aren't hiding this div, only the inner buttons have `lg:hidden`, this should be liften up to here and removed from the buttons.